README.md

```diff
@@ -27,71 +25,121 @@ Lance makes machine learning workflows with ML data easy (images, videos, point
 
 * Version, compare and diff ML datasets easily.
 
+* Search for nearest neighbors in under 1 millisecond.
+
 * (Coming soon) visualize, slice and drill-into datasets to inspect embeddings, labels/annotations and metrics.
 
 Lance is powered by Lance Format, an Apache-Arrow compatible columnar data format which is an alternative to Parquet, Iceberg and Delta. Lance has 50-100x faster query performance for ML data.
 
+
 ## Quick Start
 
+
+**Converting to Lance**
+
+```python
+import lance
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.dataset
+
+df = pd.DataFrame({"a": [5], "b": [10]})
+uri = "/tmp/test.parquet"
+tbl = pa.Table.from_pandas(df)
+pa.dataset.write_dataset(tbl, uri, format='parquet')
+
+parquet = pa.dataset.dataset(uri, format='parquet')
+lance.write_dataset(parquet, "/tmp/test.lance")
 ```
-pip install pylance
+
+**Reading Lance data**
+```python
+dataset = lance.dataset("/tmp/test.lance")
+assert isinstance(dataset, pa.dataset.Dataset)
 ```
 
-Thanks to its Apache Arrow-first APIs, `lance` can be used as a native `Arrow` extension.
-For example, it enables users to directly use `DuckDB` to analyze lance dataset
-via [DuckDB's Arrow integration](https://duckdb.org/docs/guides/python/sql_on_arrow).
+**Pandas**
+```python
+df = dataset.to_table().to_pandas()
+df
+```
 
+**DuckDB**
 ```python
-# pip install pylance duckdb
-import lance
 import duckdb
 
-# Understand Label distribution of Oxford Pet Dataset
-ds = lance.dataset("s3://eto-public/datasets/oxford_pet/oxford_pet.lance")
-duckdb.query('select class, count(1) from ds group by 1').to_arrow_table()
+tbl = dataset.to_table()  # next release of duckdb will have pushdowns enabled
+duckdb.query("SELECT * FROM tbl LIMIT 10").to_df()
 ```
 
-You can easily import a DataFrame or a Parquet file to Lance using Apache Arrow-first APIs:
+**Vector search**
+
+Download an indexed [sift dataset](https://eto-public.s3.us-west-2.amazonaws.com/datasets/sift/sift_ivf256_pq16.tar.gz),
```

Comment:
Could this be moved to wget/curl + unzip via shell? or you prefer this for general os - windows included?